project/Dependencies.scala

```diff
@@ -206,7 +207,7 @@ object Dependencies {
 
     ) ++ crossLibraryFamily("com.typesafe.play", PlayVersion)(
       "play", "play-guice", "play-java", "play-jdbc", "play-jdbc-api",
-      "play-netty-server", "play-server", "play-streams", "play-ws", "play-ahc-ws"
```

Comment:
may want to upgrade to ` val AkkaHttpVersion = "10.0.10"` at the same time

Reply:
Taken care of here: https://github.com/lagom/lagom/pull/1061/files#diff-0ecdbc5a001d52fb34f5eafb7cd1aaa6R18

This branch should rebase onto the latest `master`.